prow/config.yaml

```diff
@@ -4149,3 +4149,10 @@ periodics:
       secret:
         defaultMode: 256
         secretName: ssh-key-secret
+trigger:
+- repos:
+  - kubernetes
+  - kubernetes-incubator
```

Comment:
Is this a repo or an org? Same question for security, I thought both are orgs?
https://github.com/kubernetes-incubator
https://github.com/kubernetes-security